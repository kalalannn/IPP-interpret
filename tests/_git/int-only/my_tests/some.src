.ippcode19

# write test
write int@10
write string@some
write bool@true
write GF@so

