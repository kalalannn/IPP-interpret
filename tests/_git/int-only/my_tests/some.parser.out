<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode19">
    <instruction order="1" opcode="WRITE">
        <arg1 type="int">10</arg1>
    </instruction>
    <instruction order="2" opcode="WRITE">
        <arg1 type="string">some</arg1>
    </instruction>
    <instruction order="3" opcode="WRITE">
        <arg1 type="bool">true</arg1>
    </instruction>
    <instruction order="4" opcode="WRITE">
        <arg1 type="var">GF@so</arg1>
    </instruction>
</program>
